docs/7-deployment/1-config.md

```diff
@@ -2,3 +2,281 @@
+  ```
+
+#### S3 Storage Configuration
+
+For cloud-based core database storage:
+
+- **`CORE_DB_S3_ENDPOINT`** - S3 endpoint for cloud storage
+  ```bash
+  CORE_DB_S3_ENDPOINT=https://s3.amazonaws.com
+  ```
+
+- **`CORE_DB_S3_REGION`** - AWS region
+  ```bash
+  CORE_DB_S3_REGION=us-east-1
+  ```
+
+- **`CORE_DB_S3_KEY`** / **`CORE_DB_S3_SECRET`** - AWS credentials
+  ```bash
+  CORE_DB_S3_KEY=AKIAIOSFODNN7EXAMPLE
+  CORE_DB_S3_SECRET=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  ```
+
+- **`CORE_DB_S3_USE_SSL`** - SSL enforcement
+  ```bash
+  CORE_DB_S3_USE_SSL=true
+  ```
+
+### CORS Settings
+
+Configure Cross-Origin Resource Sharing (CORS) for web applications:
+
+- **`CORS_ALLOWED_ORIGINS`** - Permitted domains (comma-separated)
+  ```bash
+  CORS_ALLOWED_ORIGINS=http://localhost:3000,https://app.example.com
+  ```
+
+- **`CORS_ALLOWED_METHODS`** - HTTP verbs (default: GET, POST, PUT, DELETE, OPTIONS)
+  ```bash
+  CORS_ALLOWED_METHODS=GET,POST,PUT,DELETE,OPTIONS
+  ```
+
+- **`CORS_ALLOWED_HEADERS`** - Request headers
+  ```bash
+  CORS_ALLOWED_HEADERS=Content-Type,Authorization
+  ```
+
+### Authentication
+
+- **`ALLOWED_ANONYMOUS`** - Anonymous access (default: `true`)
+  ```bash
+  ALLOWED_ANONYMOUS=true
+  ```
+
+- **`ANONYMOUS_ROLE`** - Default role (default: `"anonymous"`)
+  ```bash
+  ANONYMOUS_ROLE=anonymous
+  ```
+
+- **`SECRET_KEY`** - API authentication key
+  ```bash
+  SECRET_KEY=your-secret-key-here
+  ```
+
+- **`AUTH_CONFIG_FILE`** - Configuration file path (JSON/YAML)
+  ```bash
+  AUTH_CONFIG_FILE=/config/auth.yaml
+  ```
+
+- **`ALLOWED_MANAGED_API_KEYS`** - GraphQL API key management
+  ```bash
+  ALLOWED_MANAGED_API_KEYS=true
+  ```
+
+### Cache Configuration
+
+See the [Caching](./2-caching.md) section for detailed cache configuration options.
+
+### Cluster Configuration
+
+See the [Clustered Deployment](./6-cluster.md) section for cluster-specific configuration options.
+
+## Management Node Configuration
+
+The management node coordinates synchronization across work nodes in the cluster.
+
+### General Settings
+
+- **`BIND`** - Management port (default: `:14000`)
+  ```bash
+  BIND=":14000"
+  ```
+
+- **`CLUSTER_SECRET`** - Cluster authentication (shared with work nodes)
+  ```bash
+  CLUSTER_SECRET=your-cluster-secret
+  ```
+
+- **`TIMEOUT`** - Node communication timeout (default: `30s`)
+  ```bash
+  TIMEOUT=30s
+  ```
+
+- **`CHECK`** - Node health check interval (default: `1m`)
+  ```bash
+  CHECK=1m
+  ```
+
+### OIDC Integration
+
+For authentication with OpenID Connect providers:
+
+- **`OIDC_ISSUER`** - Identity provider URL
+  ```bash
+  OIDC_ISSUER=https://accounts.google.com
+  ```
+
+- **`OIDC_CLIENT_ID`** - Application identifier
+  ```bash
+  OIDC_CLIENT_ID=your-client-id
+  ```
+
+- **`OIDC_TLS_INSECURE`** - Certificate verification bypass (not recommended for production)
+  ```bash
+  OIDC_TLS_INSECURE=false
+  ```
+
+- **`OIDC_COOKIE_NAME`** - Session cookie identifier
```

Comment:
Hugr uses the cookie name to extract token if is not provided in the Auth header